py_backend/users/views/check_user_logged_in.py

```diff
@@ -6,26 +6,7 @@
 from django.utils import timezone
 import logging
 
-logger = logging.getLogger(__name__)
-
-User = get_user_model()
-
-
-def is_user_logged_in(username):
-    try:
-        user = User.objects.get(username=username)
-    except User.DoesNotExist:
-        return False
-    
-    sessions = Session.objects.filter(expire_date__gte=timezone.now())
-    for session in sessions:
-        data = session.get_decoded()
-        if str(user.pk) == str(data.get('_auth_user_id')):
-            return True
-    return False
-
-@require_http_methods(["POST"])
+@require_http_methods(["GET"])
 @requires_csrf_token
-def check_user_logged_in_view(request, username):
-    is_logged_in = is_user_logged_in(username)
-    return JsonResponse({'is_logged_in': is_logged_in})
+def check_user_logged_in_view(request):
+    return JsonResponse({'is_logged_in': request.user.is_authenticated})
```

Comment:
A views must be return a code error, if the user is not logged in return 401 otherwise return 200